.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please gitignore your .vscode folder.

Reply:
Added .vscode to root gitignore in /Text

Reply:
you also need to remove hte files from your commit